marlowe-playground-client/src/Store.purs

```diff
@@ -0,0 +1,31 @@
+module Store where
+
+import Marlowe (class HasSPSettings, SPSettings_)
+import Type.Row (type (+))
+
+type EnvRow :: Row Type -> Row Type
+type EnvRow r
+  = ( ajaxSettings :: SPSettings_ | r )
+
+-- | Immutable part of the global state. We wrap it in a `newtype`
+-- | because we need an instance of `HasSPSettings`.
+newtype Env
```

Comment:
I agree with your comment in the PR description - I don't think this is a good idea. I'm not a fan of putting the "read-only" environment in the state like this.  I propose we have `newtype AppM = AppM (ReaderT Env (StoreT Store Aff) a)` instead. That way, the immutable environment cannot be accidentally modified.